src/dev/precommit_hook/casing_check_config.js

```diff
@@ -150,28 +150,52 @@ export const TEMPORARILY_IGNORED_PATHS = [
   'x-pack/plugins/reporting/server/export_types/common/assets/fonts/roboto/Roboto-Medium.ttf',
   'x-pack/plugins/reporting/server/export_types/common/assets/fonts/roboto/Roboto-Regular.ttf',
   'x-pack/plugins/reporting/server/export_types/common/assets/img/logo-grey.png',
+  'x-pack/test/functional/es_archives/monitoring/beats-with-restarted-instance_mb/data.json.gz',
```

Comment:
This array is intentionally designed to get smaller over time and not having things added to it, please just update these paths to stop using kebab-case.

Reply:
Makes sense, I'll update these directory names.

Reply:
* [x] Done